flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorTest.java

```diff
@@ -328,6 +330,31 @@ public void testNotNullConstraints() throws Exception {
 		}
 	}
 
+	@Test
+	public void testTimestamp() throws Exception {
```

Comment:
Please add tests to `testDifferentFormats` too, because some formats like orc are total another way.

Reply:
Added test in `testDifferentFormats ` but there's some issue with the vectorized ORC reader, so I temporarily turned off the reader. Please have a look.

Reply:
Please not turn off whole reader. You can modify `HiveTableInputFormat.useOrcVectorizedRead`.